Polyhedron/demo/Polyhedron/Three.cpp

```diff
@@ -130,3 +128,16 @@ void Three::autoConnectActions(Polyhedron_demo_plugin_interface *plugin)
              action->objectName().toUtf8().data());
   } // end foreach action of actions()
 }
+
+void Three::s_information(QString s)
```

Comment:
s stands for static, and I added it to avoid conflicts with the MainWindow implementation, as it inheritates Three and Message_interface. I figured it was better if the names differed.

Reply:
Yet those names are ugly. We need to find a solution.

Reply:
we could rename the ones from message_interface, as they shouldn't appear anymore. Something like message_warning() etc. And keep warning() for the static functions of Three.

Reply:
That is a good solution. Thanks.